src/app/components/navbar/navbar.component.sass

```diff
@@ -42,3 +42,25 @@
 
 .settings-link:visited
     color: var(--text-light-primary, white)
+
+.download-card
+    width: 400px
+    box-shadow: 0
+    border-radius: 0
+    border: 2px solid var(--border-color)
+    border-top: 0
```

Comment:
This effectively removes any visual styles that make up a card (see comment about the usage of mat card for repeating list like content above).